canonical-web-and-design/jaas-dashboard
# Upgrade jujulib to 2.0.0-beta.3

## Done

jujulib upgraded to resolve `logout` issue.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Use the app a bit, it should continue to work, refresh it a bunch of times to see if you can get a model connection timeout, it should keep working.
